wolfcrypt/src/evp.c

```diff
@@ -9465,6 +9483,7 @@ int wolfSSL_EVP_PKEY_assign_DH(EVP_PKEY* pkey, WOLFSSL_DH* key)
 
 
 
+#endif /* OPENSSL_EXTRA */
```

Comment:
Two OPENSSL_EXTRA only #ifdefs?
Seems that everything from Line 36-9468 (in original file) is only compiled with OPENSSL_EXTRA defined.

Reply:
The second endif comment should read "OPENSSL_EXTRA || HAVE_CURL". In that commit was keeping the functions in the same location and HAVE_CURL just compiled the MD digest parts. Will add a commit re-aranging the functions and grouping the MD digest ones together (harder to apply a patch of though to an older version of wolfSSL)